ProcessMaker/processmaker
# Apply changes for patch 2 (4.3.0-patch-2-les-schwab): performance improvements, node ssr, previous patch changes,

## Issue & Reproduction Steps
This PR will represent the changes necessary to create patch v2 for v4.3.0 for a ProcessMaker client as defined in [RM-1534](https://processmaker.atlassian.net/browse/RM-1534). This is to ensure prior changes will be included in the new patch when applied.

## Solution(s)
#### Updated in Core:
- Merged [`bugfix/FOUR-7193`](https://github.com/ProcessMaker/processmaker/pull/4591) into `4.3.0-patch-2-les-schwab-branch`
- Merged [`bugfix/FOUR-8342`](https://github.com/ProcessMaker/processmaker/pull/4735) into `4.3.0-patch-2-les-schwab-branch`
- Merged [`bugfix/FOUR-8349`](https://github.com/ProcessMaker/processmaker/pull/4730) into `4.3.0-patch-2-les-schwab-branch`
- Merged [`bugfix/FOUR-8347`](https://github.com/ProcessMaker/processmaker/pull/4732) into `4.3.0-patch-2-les-schwab-branch`

#### Updates Packages
- Updated `processmaker/docker-executor-node-ssr` to version [`1.3.0`](https://github.com/ProcessMaker/docker-executor-node-ssr/pull/7)
- Update `processmaker/package-data-sources` to branch found on related PR [`1.11.3-les-schwab-patch`](https://github.com/ProcessMaker/package-data-sources/pull/273)
- Update `processmaker/package-savedsearch` to branch found on related PR [`1.19.2-les-schwab-patch`](https://github.com/ProcessMaker/package-savedsearch/pull/353)

## How to Test
- Visit each related PR for the above merges/new package versions and follow the "_How to Test_" section.
- Ensure all email, authentication, PDF/screen rendering, script tasks, telemetry (DataDog), and process request permissions are functioning as expected

## Related Tickets & Packages
- HELP ticket: https://processmaker.atlassian.net/browse/PM4EH-381
- New patch request: https://processmaker.atlassian.net/browse/RM-1538
- Original patch request: https://processmaker.atlassian.net/browse/RM-1534

## Code Review Checklist

## CircleCi

### Package Versions
ci:connector-docusign:v1.1.0
ci:connector-pdf-print:v1.8.1
ci:connector-send-email:v1.12.0
ci:connector-slack:v1.3.0
ci:docker-executor-node-ssr:v1.3.0
ci:package-actions-by-email:v1.7.1
ci:package-advanced-user-manager:v1.1.0
ci:package-auth:v1.6.0
ci:package-collections:v2.6.1
ci:package-comments:v1.4.0
ci:package-conversational-forms:v1.2.2
ci:package-data-sources:1.11.3-les-schwab-patch
ci:package-dynamic-ui:v1.5.0
ci:package-files:v1.5.0
ci:package-googleplaces:v1.3.0
ci:package-process-documenter:v1.2.0
ci:package-process-optimization:v1.3.0
ci:package-savedsearch:1.19.2-les-schwab-patch
ci:package-sentry:v1.3.0
ci:package-signature:v1.4.0
ci:package-translations:v2.3.1
ci:package-versions:v1.3.0
ci:package-vocabularies:v2.7.0
ci:package-webentry:v2.6.2

#### Deployment
ci:deploy
 

[RM-1534]: https://processmaker.atlassian.net/browse/RM-1534?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
